### PR TITLE
ci: adds release notes generation

### PR DESCRIPTION
**Pull request by benventive**
GitHub Issue: #

## PR Type

What kind of change does this PR introduce?

- Build or CI related changes

## Description

Adds the release notes generation step. This generates a simple markdown file with information about the commit, the branch, the build pipeline and the application environment.
This PR also includes a fix for the AppCenter release for Android.

## PR Checklist 
Please check if your PR fulfills the following requirements:

- [ ] Interface members are XML documented
- [ ] Documentation (XML or comments) has been added and/or existing documentation has been updated
- [ ] [Architecture documents](./doc/Architecture.md) have been updated
- [ ] Tested on all relevant platforms

### CODE REVIEW

File: build/steps-build.yml [hunk 1]
```diff
@@ -112,6 +112,15 @@ steps:
     codeCoverageTool: 'Cobertura'
     summaryFileLocation: "$(Build.SourcesDirectory)/src/coverage.cobertura.xml"
 
+- task: nventiveReleaseNotesCompiler@5
```
Comment (jeanplevesque) [inline build/steps-build.yml:115]: Is there doc for this task somewhere?

**Review by jeanplevesque — APPROVED** [review]

File: build/steps-build.yml (hunk 1 above)
Comment (benventive) [reply build/steps-build.yml:115]: Not really no, I can make something in the repository! It's really basic, that'll be quick

Comment (jeanplevesque) [reply build/steps-build.yml:115]: I think comments right there would be fine

Comment (jeanplevesque) [inline build/steps-build.yml:115]: Is this task public? Is it going to work for people outside the nventive organization?

Comment (benventive) [reply build/steps-build.yml:115]: No, it's not. It could easily be made pubic though, given its simplicity.

**Pull request merged**